vllm/multimodal/inputs.py

```diff
@@ -96,7 +96,8 @@ class PlaceholderRange(TypedDict):
     """The length of the placeholder."""
 
 
-NestedTensors = Union[List["NestedTensors"], List[torch.Tensor], torch.Tensor]
+NestedTensors = Union[List["NestedTensors"], List[torch.Tensor], torch.Tensor,
+                      Tuple[torch.Tensor]]
```

Comment:
What does `Tuple[torch.Tensor]` mean here? Is it really a tuple with a single tensor?

Reply:
Fixed